payment-account-age-witness.adoc

```diff
@@ -0,0 +1,196 @@
+== Payment account age witness
+:toc:
+
+....
+  Bisq-IP: 2
+  Layer: Exchange
+  Title: Payment account age witness
+  Author: Manfred Karrer <mk@nucleo.io>
+  Comments-Summary: No comments yet.
+  Comments-URI: Bisq Forum thread [1]
+  Status: Draft
+  Created: 2017-09-14
+  License: GPL Version 3
+....
+
+== Abstract
+
+This proposal describes a verification model for the age of the first trade with a certain payment account as protection against a fraud scheme in which a criminal has obtained illegal access of a foreign bank account and tries to buy Bitcoin with the stolen funds.
+
+The victim of the stolen bank account will likely contact his bank and initiate a bank chargeback.
+The Bitcoin seller would in such a case be at risk to lose the received payment in Fiat currency.
+
+We *assume* that the criminal who has access to the bank account is intending to take out the funds of that account as quickly as possible as well as that he intends to do that in few large transactions because with each transaction the risk increases that the fraud gets discovered and the account gets frozen.
+
+With our *trade amount limit* we have already a partial protection against that fraud scheme but we would like to increase the security by having additionally a verification scheme for the *age of the bank account*.
+
+To protect users privacy we use a hashing scheme and only the other trading peer who will receive anyway the payment details during the trade process is able to verify that the provided hash in the offer matches the real account data.
```

Comment:
and only the other trading peer - who will receive the payment details anyway - is able to verify ...